pkg/web_app/lib/src/account.dart

```diff
@@ -345,17 +355,29 @@ class _PublisherAdminWidget {
     );
   }
 
-  Future<void> _inviteMember() async {
+  Future<void> _addMember() async {
+    await modalWindow(
+      titleText: 'Invite new member',
+      isQuestion: true,
+      okButtonText: 'Add',
+      content: _addMemberContent,
+      onOkButtonClicked: () => _inviteMember(false),
+    );
+  }
+
+  Future<bool> _inviteMember(bool requestConfirm) async {
```

Comment:
instead of introducing `requestConfirm` what about making two methods `_inviteMember` and `_inviteMemberExperimental`, then it's easy to remove `_inviteMember` and rename `_inviteMemberExperimental` when the experimental part goes away.

Reply:
Added a `TODO` instead. I think duplicating the method for the 3-4 lines of difference is not worth the risk of fixing something in one and missing it in the other. When `requestConfirm` is removed, it will be a clean update, it is used only in one conditional expression.